api/capi/src/tensor_filter_single.c

```diff
@@ -276,6 +311,9 @@ g_tensor_filter_single_invoke (GTensorFilterSingle * self,
 
   GST_TF_FW_INVOKE_COMPAT (priv, status, input, output);
 
+  if (self->allocate_in_invoke) {
+  }
```

Comment:
Mistake? Otherwise, please add some comments like `TODO`

Reply:
This is a mistake. Removed.